features/topology-map/plugins/org.opennms.features.topology.plugins.topo.linkd/src/test/java/org/opennms/features/topology/plugins/topo/linkd/internal/LinkdTopologyProviderTestIT.java

```diff
@@ -143,6 +152,12 @@ public void testOspf() throws Exception {
         test(TopologyGenerator.Protocol.ospf);
     }
 
+    @Test
+    @Transactional
+    public void testBridgeBridge() throws Exception {
+        test(TopologyGenerator.Protocol.bridgeBridge);
```

Comment:
Can we add more specific tests? i.e. verifying that specific nodes are connected via a given shared segment

Reply:
yes will do once we are able to actually generate the topology - unfortunately I am not there yet...

Reply:
done.